### DISCUSSION TITLE
网络正常的情况下，应用无法联网

### DISCUSSION THREAD

**Discussion by Bradford-Chen** [Richasy/Bili.Uwp#817] [问答]
<img width="568" alt="image" src="https://user-images.githubusercontent.com/56224923/157361338-7decd57f-2a51-458c-94cb-cf4880d2d82c.png">
设置中没有”选择能使用你的 WLAN 数据的应用“这个选项
<img width="605" alt="image" src="https://user-images.githubusercontent.com/56224923/157361450-ec6bfe38-9018-4efe-a750-c25610360d61.png">

**Comment by n0789** [comment]
建议看一下这篇 Microsoft Community 帖子: 
https://answers.microsoft.com/zh-hans/windows/forum/all/%E9%80%89%E6%8B%A9%E8%83%BD%E4%BD%BF%E7%94%A8/b4a94101-b2b7-482a-bf93-86052f535d31

**Comment by Richasy** [comment]
你是否打开了代理？你已经看到文档的那里，不妨试试按照开代理的那一部分的解决方法操作
